internal/server/routes.go

```diff
@@ -47,7 +47,7 @@ func (s *Server) GenerateRoutes(promRegistry prometheus.Registerer) Routes {
 	// This group of middleware will apply to everything, including the UI
 	router.Use(
 		logging.Middleware(),
-		TimeoutMiddleware(1*time.Minute),
```

Comment:
Did you mean to change this to 2 minutes, or was this for debugging? Might be good to make this in a separate PR if you did mean to change it.

Reply:
I found it annoying to debug to step through the code in less than a minute and figured 2 minutes was still a reasonable timeout for everyone. can remove it